gradle.properties

```diff
@@ -10,7 +10,7 @@ sizeofVersion = 0.3.0
 jaxbVersion = 2.3.1
 
 # Terracotta clustered
-terracottaPlatformVersion = 5.8.0
+terracottaPlatformVersion = 5.8-SNAPSHOT
```

Comment:
requires https://github.com/Terracotta-OSS/terracotta-platform/pull/861/ and a new release